### ISSUE TITLE
Using LightGallery with Angular 13

### ISSUE THREAD

**Issue by dhlevi** [sachinchoolur/lightGallery#1395]
## Description

When updating to Angular 13, seeing the following error:

`static ɵprov: i0.ɵɵInjectableDef<LightgalleryService>;`

### Steps to reproduce

Use lightGallery with Angular 13

### Environment

-   lightGallery version - 2.6.0

### Additional context

Not really a bug directly... Aside from doing a local update/build for lightGallery, is there a plan for updating the component to support Angular 13?

**Comment by sachinchoolur** [comment]
Hey @dhlevi,

Working on creating new components for angular 13 and 14.

**Comment by sachinchoolur** [comment]
Angular 13 and 14 components are available in version 2.7.0-beta.0

For Angular 13, you need to import the lightGallery component using `lightgallery/angular/13`

``` js
import { LightgalleryModule } from 'lightgallery/angular/13';
```

let me know if you face any issues

**Comment by dhlevi** [comment]
Thanks @sachinchoolur, working like a charm so far, but I'll let you know if we experience anything unexpected.
